File: GumiPresso_Server/src/main/java/com/ssafy/through/model/service/BannerService.java
package com.ssafy.through.model.service;

import java.util.List;

import com.ssafy.through.model.dto.Banner;

public interface BannerService {
	int insertBanner(Banner banner);
	List<Banner> selectBanner();
	int updateBanner(Banner banner);
	int deleteBanner(Banner banner);
}

File: GumiPresso_User/app/src/main/java/com/ssafy/gumipresso/model/service/BannerService.kt
package com.ssafy.gumipresso.model.service

import com.ssafy.gumipresso.model.dto.Banner
import retrofit2.Response
import retrofit2.http.GET

interface BannerService {
    @GET("/banner/")
    suspend fun getBannerList(): Response<List<Banner>>
}